tla/consensus/Traceccfraft.tla

```diff
@@ -522,6 +522,9 @@ TraceMatchesConstraints ==
     /\ MembershipStateConsistentInv
     /\ CommitCommittableIndices
 
+CommittableIndicesIsConstantSpaceInv ==
+    "committable_indices" \in DOMAIN logline.msg.state => Len(logline.msg.state.committable_indices) <= 2
```

Comment:
Does this need the implication? `committable_indices` doesn't seem to be optional in the json logging in the diff above.

Reply:
It's not on "drop_pending_to" at the moment. It looks like there is another problem though.